docs/marketplace/fastlane/testing-distribution/index.md

```diff
@@ -141,6 +141,6 @@ Utilize environment variables seamlessly by substituting the parameters with `EN
 
 ## References
 
-- For details on generating an Appcircle Personal API Token, visit [Generating/Managing Personal API Tokens](/appcircle-api-and-cli/api-authentication#generatingmanaging-the-personal-api-tokens).
+- For details on generating an Appcircle Personal API Token, visit [Generating/Managing Personal API Tokens](/account/my-organization/security/personal-api-token).
```

Comment:
This usage is repeated in many places.
Can you turn it into a file and import it into the places where it needs to be repeated?
Just like in this file:
- https://github.com/search?q=repo%3Aappcircleio%2Fappcircle-docusaurus%20_publish-steps-runner-usage-caution.mdx&type=code
- https://github.com/appcircleio/appcircle-docusaurus/blob/develop/docs/_publish-steps-runner-usage-caution.mdx